accounts-db/benches/read_only_accounts_cache.rs

```diff
@@ -0,0 +1,287 @@
+#![feature(test)]
+
+extern crate test;
+
+use {
+    criterion::{criterion_group, criterion_main, BenchmarkId, Criterion},
+    rand::{rngs::SmallRng, seq::SliceRandom, SeedableRng},
+    solana_accounts_db::{
+        accounts_db::AccountsDb, read_only_accounts_cache::ReadOnlyAccountsCache,
+    },
+    std::{
+        sync::{
+            atomic::{AtomicBool, Ordering},
+            Arc,
+        },
+        thread::Builder,
+        time::{Duration, Instant},
+    },
+};
+mod utils;
+
+/// Numbers of reader and writer threads to bench.
+const NUM_READERS_WRITERS: &[usize] = &[
+    8,
+    16,
```

Comment:
I will also uncomment them. Now the 32 and 64 cases run absolutely fine on my personal machines. The reason they were freezing them before is that we were doing N (our iterations) * M (criterion's iterations) stores (because we had a loop inside `iter`/`iter_batched`). Now with `iter_custom` and limiting the number of stores to the `iter` value provided by criterion, the bench is more efficient.